lib/middleware/permissions.js

```diff
@@ -1,11 +1,14 @@
-const { noop } = require('lodash');
+const { noop, some, castArray, without } = require('lodash');
 const { UnauthorisedError } = require('../errors');
 
-module.exports = (task, params = noop) => (req, res, next) => {
-  if (req.permissionHoles && req.permissionHoles.includes(task)) {
+module.exports = (tasks, params = noop) => (req, res, next) => {
+  tasks = castArray(tasks);
+  if (req.permissionHoles && !without(tasks, ...req.permissionHoles).length) {
     return next();
   }
-  req.user.can(task, { ...req.params, ...params(req, res) })
-    .then(can => can ? next() : next(new UnauthorisedError()))
+  Promise.all(
+    tasks.map(task => req.user.can(task, { ...req.params, ...params(req, res) }))
+  )
+    .then(cans => some(cans) ? next() : next(new UnauthorisedError()))
```

Comment:
I think this is the answer to my question. It only requires one permission to be good?

Reply:
yeah - it's because an NTCO cannot update, but can review, so you either need update or review permissions